diff --git a/dist/shared/unimport.Ln-jZm4m.mjs b/dist/shared/unimport.Ln-jZm4m.mjs
index 98760fc43dd9ebaeccea2dae658463c4ff441d97..9ad90a7b41d65850d355bc656136e9a8664e85b7 100644
--- a/dist/shared/unimport.Ln-jZm4m.mjs
+++ b/dist/shared/unimport.Ln-jZm4m.mjs
@@ -530,6 +530,14 @@ function* findDirective(imports, symbol, [begin, end, importName]) {
     } else {
       resolvedName = kebabCase(i.as ?? i.name);
     }
+    if (resolvedName === importName) {
+      yield [
+        begin,
+        end,
+        { ...i, name: i.name, as: symbol }
+      ];
+      return;
+    }
     if (resolvedName[0] === "v") {
       resolvedName = resolvedName.slice(resolvedName[1] === "-" ? 2 : 1);
     }
